### ISSUE TITLE
ENV config overrides trouble.

### ISSUE THREAD

**Issue by itbrandonsilva** [gentics/mesh#326]
## Gentics Mesh Version
- v0.17.x

## Operating System
- Linux (Ubuntu 16.04)

## JVM
- Open JDK 1.8.0_151

## Problem

Based on [this doc](https://getmesh.io/docs/beta/administration-guide.html#_environment_variables) I should be able to override corresponding mesh.yml properties by setting env variables. Here are my variables:

```
brandon@brandon-gentics:~$ echo "$MESH_HTTP_CORS_ORIGIN_PATTERN"
*
brandon@brandon-gentics:~$ echo "$MESH_HTTP_CORS_ENABLE"
true
```

After starting up mesh with these variables present, cors seems to not be enabled. There are no issues with configuring the yaml file manually with these 2 values and restarting mesh.

## Expected behaviour and actual behaviour

I expect to be able to override the yaml configuration using environment variables, but the actual behavior is that I can't.

**Comment by Jotschi** [comment]
@itbrandonsilva Thanks for the report. The fix for the bug will be shipped with the next release. (next week)

**Comment by Jotschi** [comment]
@itbrandonsilva Fix has been released (0.17.1)
